src/main/java/seedu/address/ui/PersonListPanel.java

```diff
@@ -52,6 +59,33 @@ private void setEventHandlerForSelectionChangeEvent() {
     }
 
     /**
+     * Function to increase all person cards' font sizes in person list
+     */
```

Comment:
Coding standard violation:exclamation: The first statement of the header comment does not follow the prescribed style. See point 4, 3rd bullet point of the [[Java Coding Standard: Comments](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments)]

Reply:
I see. So it should state the action immediately instead of saying X does Y. Thanks. Will work on it.